### PR TITLE
Document policy for stalled pull requests

### PR DESCRIPTION
**Pull request by sevein**
Connects to https://github.com/archivematica/Issues/issues/1435.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ Archivematica itself.
   - [Opening the pull request](#opening-the-pull-request)
   - [Discussion](#discussion)
   - [Cleaning up the commit history](#cleaning-up-the-commit-history)
+  - [Stalled pull requests](#stallel-pull-requests)
```
Comment (tw4l) [inline CONTRIBUTING.md]:
```suggestion
  - [Stalled pull requests](#stalled-pull-requests)
```

**Review by tw4l — APPROVED** [review]
I pointed out one typo but otherwise LGTM :muscle:

File: CONTRIBUTING.md (hunk 1 above)
Comment (sevein) [reply CONTRIBUTING.md]: Thanks!

**Review by ablwr — APPROVED** [review]

**Review by joel-simpson — APPROVED** [review]
looks good to me too!

**Pull request merged**